gitops-semver-increment-yaml/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.13.5
+
+RUN apk add yq nodejs --no-cache 
```

Comment:
Worth mentioning that the package repos for alpine 3.13, ships the 3.x version of `yq`. 

The v4 release is [quite different](https://mikefarah.gitbook.io/yq/v/v4.x/upgrading-from-v3) to the extent that the previous commands no longer at all. 

I don't know if it's worth downloading `yq` directly instead of using the distro version, since at some point I imagine we'll have to update anyway.

Reply:
Maybe use a node-alpine image so it's one fewer thing to install?  Or is there a reason to install it here?  Probably just fine like this though.